projectile-export/src/main/scala/com/kyleu/projectile/models/feature/core/thrift/ThriftServiceFile.scala

```diff
@@ -10,28 +10,85 @@ import com.kyleu.projectile.models.thrift.input.{ThriftFileHelper, ThriftMethodH
 object ThriftServiceFile {
   def export(config: ExportConfiguration, svc: ExportService) = {
     val file = ScalaFile(path = OutputPath.ServerSource, dir = svc.pkg, key = svc.className)
+    val classCanonicalName = (svc.pkg :+ svc.className).mkString(".")
 
     file.addImport(Seq("scala", "concurrent"), "Future")
     val thriftService = svc.pkg.dropRight(1) :+ svc.key
 
     config.addCommonImport(file, "TraceData")
+    config.addCommonImport(file, "TracingService")
+
     config.addCommonImport(file, "ThriftFutureUtils", "toScalaFuture")
     config.addCommonImport(file, "ThriftService")
-    config.addCommonImport(file, "ThriftServiceHelper")
     file.addImport(Seq("com", "twitter", "finagle"), "ThriftMux")
     file.addImport(Seq("com", "twitter", "scrooge"), "Request")
 
     val thriftServiceCanonicalName = thriftService.mkString(".")
+    val thriftReqRepServicePerEndpointCanonicalName = s"$thriftServiceCanonicalName.ReqRepServicePerEndpoint"
+
+    file.add(
+      s"""
+         |/***
+         |* {{{
+         |* val svcPerEndpoint = $classCanonicalName.mkServicePerEndpoint(url)
+         |* val myService = $classCanonicalName
+         |*   .from(svcPerEndpoint)
+         |*   .withTracingService(openTracingTracingService)
+         |*   .withThriftSpanNamePrefix("myThriftService")
+         |*   .withTraceDataSerializer(td => Map.empty)
+         |* }}}
+         |*/
+       """.stripMargin)
```

Comment:
Doesn't this screw with the indention?

Reply:
It works fine, at least for the javadoc comment.

Reply:
The auto-formatter will rewrite it, which causes every compile to rewrite files twice, once for codegen, once formatted. I'll merge it, fix the formatting, and publish a new version. Thanks for the contribution